app/views/search/show.html.erb

```diff
@@ -9,8 +9,8 @@
 <br>
 <% @movies.results.each do |movie| %>
   <p class="movie">
-    Movie Title: <%= link_to "#{movie[:original_title]}", "/movies/#{movie[:id]}" %><br>
+    Movie Title: <%= link_to "#{movie[:original_title]}",  movie_details_path('uuid') %><br>
```

Comment:
I believe 'movie' would be the more restful prefix for the movies show page (rather than 'movie_details')